src/BannerWithLink/styles.ts

```diff
@@ -52,16 +46,32 @@ export const styles = {
             font-size: 20px;
             max-width: 37rem;
         }
-
-        ${from.wide} {
-            max-width: 42rem;
-        }
     `,
     cta: css`
-        font-weight: 700;
+        color: ${neutral[0]};
         margin-top: ${space[3]}px;
-        margin-right: ${space[3]}px;
-        display: inline-block;
-        color: ${neutral[20]};
     `,
+    closeButton: css`
+        &:hover {
+            background-color: transparent;
+        }
+    `,
+    infoIcon: css`
+        svg {
+            fill: ${neutral[0]};
+            background: transparent;
+        }
+    `,
+    heading: css`
+        color: ${neutral[0]};
+    `,
+};
+
+// Use this theme but override the colours to match the moment colours
+export const overrridenReaderRevenueTheme: { button: ButtonTheme } = {
```

Comment:
I remember this was used to make a button not change colour when depending on source for the first time (or a source upgrade or something similar). Should we create a ticket to follow up on removing this code? Eventually the palette that marketing want to use for Braze components should live in Source, instead of us making little tweaks to the existing palettes.

Reply:
Yeah agreed, hopefully this can be removed once we figure out how to handle palettes for the banner. (Just to note: this isn't an addition in this PR, it just moved from the main component index file). Though you pointing this out does make me realise there's a tiny typo in the const name here so will fix that!